packages/utils/upgrade/src/core/project-loader.ts

```diff
@@ -81,19 +82,19 @@ const loadProjectFiles = async (options: ProjectLoaderOptions): Promise<string[]
     options.allowedExtensions ?? PROJECT_DEFAULT_ALLOWED_EXTENSIONS
   );
 
-  const pattern = `./${allowedRootPaths}/**/*.${allowedExtensions}`;
+  const projectFilesPattern = `./${allowedRootPaths}/**/*.${allowedExtensions}`;
+  const patterns = [projectFilesPattern, ...PROJECT_DEFAULT_PATTERNS];
 
-  const files = await new Promise<string[]>((resolve, reject) => {
-    glob(pattern, { cwd }, (err, matches) => (err ? reject(err) : resolve(matches)));
-  });
+  const files = await glob(patterns, { cwd });
 
   const fFilesLength = f.highlight(files.length.toString());
-  const fPattern = f.highlight(pattern);
+  const fPattern = f.highlight(patterns.map((p) => `"${p}"`).join(', '));
```

Comment:
`"` is a valid character in paths, so we may need to escape it (or tbh if it's more complicated than expected we could just document that the allowedRootPaths must be an escaped value, it's not a common use case)

Reply:
fPattern is only for logging purpose though, does it still make sense to take the escaping in consideration?